lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -114,10 +119,13 @@ class TurboGraft.Remote
       initiator: @initiator
       xhr: xhr
 
-    if @refreshOnError || @refreshOnErrorExcept
```

Comment:
Yes, although in practice it's unlikely anyone would have noticed. In turbolinks.coffee, in changePage, if there are only keys, then we don't use the expect keys. What that meant, is that if you had both, we never used the expect keys. In most cases, the section which would be refreshed on error would not overlap the section to not refresh on error, so it wouldn't really matter, i.e. by only refreshing the on refresh keys, you would as a side effect not refresh the except keys. My opinion is that by changing this around, into separate if statements, we make the behaviour clearer and more explicit.